src/main/kotlin/org/rust/ide/colors/RsColors.kt

```diff
@@ -60,6 +60,8 @@ enum class RsColor(humanName: String, default: TextAttributesKey? = null) {
     DOC_COMMENT("Rustdoc//Comment", Default.DOC_COMMENT),
     DOC_HEADING("Rustdoc//Heading", Default.DOC_COMMENT_TAG),
     DOC_LINK("Rustdoc//Link", Default.DOC_COMMENT_TAG_VALUE),
+    DOC_EMPHASIS("Rustdoc//Emphasis"),
+    DOC_STRONG("Rustdoc//Emphasis strong"),
```

Comment:
Could you add the corresponding examples into `highlighterDemoText.rs` to show examples in color setting page, please?